src/main/java/RWSet.java

```diff
@@ -1,95 +1,68 @@
 import java.util.Set;
+import java.util.ArrayList;
 import java.util.HashSet;
 
 /**
  * Remove Wins Set
  */
-public class RWSet implements CRDT {
+public class RWSet<T extends Object> extends CRDT {
```

Comment:
p. sure all Ts extend Object

Reply:
I ended up taking this out since I believe it's a non-issue. May as well allow T's which for some reason don't extend the java lang object